src/libs/actions/Report.ts

```diff
@@ -574,6 +562,7 @@ function reportActionsExist(reportID: string): boolean {
  */
 function openReport(
     reportID: string,
+    reportActionID?: string,
```

Comment:
For comment linking, we need to be able to open a report with a specific range of report actions. When we pass the reportActionID, we can select a specific range around this report action

Reply:
Ah, now I understand what you mean.
I made this change because we usually don't use more than two parameters in the code. To be honest, I prefer using destructuring there. It would make things clearer.
If you insist, I can change it as you suggested.
![image](https://github.com/Expensify/App/assets/48593211/ed446636-5198-4b05-8d4d-f3a54b077425)